source/meetup-assets.html.erb

```diff
@@ -108,55 +108,50 @@ responsive: true
-    </li>
-    <li>
-      —When everything's ready to go, hit the record button on the capturing laptop, hit the record button on the iPhone, hit the record button on the voice recorder, and you're off to the races.
-    </li>
-    <li>
-      —After the event, download all three files onto one laptop, and use something like Screenflow to sync them up and produce your video. Alternatively, if you have <em>some</em> sponsorship, you can find a local producer or videographer to do this work for you for a few hundred dollars, depending on the volume of content.
-    </li>
-  </ul>
-</p>
-
-<h1>Organizers Groups</h1>
   <p>
-    There's an infrequently-used but future-helpful group of Ember Meetup Organizers, which we use for organizer chit chat and announcements. <a href="https://groups.google.com/forum/#!forum/ember-meetup-organizers">You can apply to join the group here</a>. Be sure to list what group you're a part of in your application to ensure acceptance.
+    To make this setup work, you'll need a laptop, in <em>addition</em> to the presenter's laptop. The basic process is:
+    <ul>
+      <li>
+        The presenter's laptop gets plugged via HDMI into the Video Capture device (HDMI In).
+      </li>
+      <li>
+        The video capture device gets plugged via HDMI into the projector (HDMI Out).
+      </li>
+      <li>
+        The video capture device <em>also</em> plugs into the extra laptop via USB.
+      </li>
+      <li>
+        The presenter wears the Voice Recording device in a pocket, with the Lavalier mic plugged in and attached somewhere near their neck. This will record your separate, clear audio stream.
+      </li>
+      <li>
+        The tripod is positioned somewhere with a good view of the speaker, with the phone-specific-attachment screwed in, and then the phone inserted horizontally.
+      </li>
+      <li>
+        When everything's ready to go, hit the record button on the capturing laptop, hit the record button on the iPhone, hit the record button on the voice recorder, and you're off to the races.
+      </li>
+      <li>
+        After the event, download all three files onto one laptop, and use something like Screenflow to sync them up and produce your video. Alternatively, if you have <em>some</em> sponsorship, you can find a local producer or videographer to do this work for you for a few hundred dollars, depending on the volume of content.
+      </li>
+    </ul>
   </p>
 
+  <h1>Organizers Network</h1>
   <p>
-    There's also an Ember Meetup Organizer Slack Channel, which we're happy to add new organizers to. <a href="mailto:meetups@emberjs.com">Reach out via email to be added</a>, and be sure to specify what group you're working on.
+    There's a great (not-so-secret) channel in the Ember Community Discord for organizers. Request to be added to the <dfn>meetup-organizer</dfn> role in #discord-server-admin so that you can join us!
```

Comment:
Would it make sense to add a link for the Discord invite?

```suggestion
    There's a great (not-so-secret) channel in the <a href="https://discordapp.com/invite/zT3asNS">Ember Community Discord</a> for organizers. Request to be added to the <dfn>meetup-organizer</dfn> role in #discord-server-admin so that you can join us!
```

Reply:
Yes, I think so :) I just did not know if there was any! Thanks